### DISCUSSION TITLE
Icon Genie v2 Error

### DISCUSSION THREAD

**Discussion by tacot9** [quasarframework/quasar#11520] [CLI - SSR mode]
I ran the code
`icongenie generate -i/mypath/myimage.png`
and received the error, "Error. This command must be executed inside a Quasar v1+ project folder."
I then tried running 
`npm r -g @quasar/app`
`npm i -g @quasar/cli`
No error code showed. I proceeded to run the icon genie generate command again and it threw the same error.
This is what my quasar info shows.
`NodeJs                          14.18.0
Global packages
  NPM                           6.14.15
  yarn                          Not installed
  @quasar/cli                   1.2.2
  @quasar/icongenie             2.4.2
  cordova                       Not installed`

Anyone have any idea what is going on?

**Comment by hawkeye64** [comment]
@tacot9 Run it in the same folder as `quasar.conf.js`
Make sure you have installed the node_modules, too
